### DISCUSSION TITLE
请联系我领取 EasyWeChat 贡献奖品

### DISCUSSION THREAD

**Discussion by overtrue** [w7corp/easywechat#2149] [General]
@Abbotton @bangbangda @moniang @millsguo @xyj2156 @sunmingyang @xmjharry @ljyljy0211 @fireqong

恭喜以上同学在 EasyWeChat 打榜赛中获奖，感谢你们的贡献，请回复此贴收件地址领取奖品，格式：**姓名，手机，地址**

> 请评论完地址后10秒钟内删除 （评论后我会收到邮件）

**Comment by moniang** [comment]
打榜赛是什么？

**Reply by overtrue** [reply]
https://wiki.w7.cc/chapter/1309?id=3750&utm_source=easywechat.com

**Comment by overtrue** [comment]
赛事说明：https://wiki.w7.cc/chapter/1309?id=3750

**Comment by overtrue** [comment]
请评论完地址后10秒钟内删除或打码

**Comment by Abbotton** [comment]
想当年因为自己的原因在微擎入职后只上了一天班，现在想起还真是有点遗憾。

**Reply by overtrue** [reply]
哈哈哈

**Comment by overtrue** [comment]
请以上同学速度回复哦，不然后天过期就算放弃了

**Reply by TheNorthMemory** [reply]
放弃的咋处理呀，😄

**Reply by overtrue** [reply]
应该就是不给咯哈哈哈

**Comment by Abbotton** [comment]
超哥，能否公布一下还没有回复的同学😄?

**Comment by overtrue** [comment]
@ljyljy0211
@fireqong
@sunmingyang
@millsguo
@bangbangda

这几位同学没回复哈，其它已经收到不需要重复回复了。

**Reply by fireqong** [reply]
已回复，超哥

**Reply by fireqong** [reply]
我这是不是有点晚了。。。

**Reply by fireqong** [reply]
感觉错过了一个亿，你看我还有机会么。。

**Reply by overtrue** [reply]
没过期

**Comment by overtrue** [comment]
请一下同学 7 月 20 日前回复收件信息，逾期则表示放弃领取，谢谢！
@ljyljy0211
@sunmingyang
@millsguo
@bangbangda

**Reply by bangbangda** [reply]
已回复

**Reply by overtrue** [reply]
@bangbangda 收到

**Reply by sunmingyang** [reply]
已回复，感谢

**Comment by overtrue** [comment]
请以下同学 7 月 20 日前回复收件信息，逾期则表示放弃领取，谢谢！
@ljyljy0211
@millsguo

**Comment by overtrue** [comment]
已截止

**Reply by ZhangChengLin** [reply]
啥奖品，咋没人要嘞？

**Comment by ljyljy0211** [comment]
啊，我来迟了，咋办，今天好像是最后一天，还有机会吗

**Reply by ljyljy0211** [reply]
已收到通知了，3Q

**Comment by overtrue** [comment]
已结束，颁奖结果：https://wiki.w7.cc/chapter/1309?id=4724
